src/muiComponents/Input/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { ReactElement } from "react"
+import { Box } from "@mui/material"
+
+interface Props {
+  token: number
+}
+
+export default function TokenInput({ token }: Props): ReactElement {
+  return (
+    <div>
```

Comment:
Is outer `div` wrap necessary?

Reply:
This component didn't finished yet. Might be I'll work on another branch.